scripts/setup/setup.cfg

```diff
@@ -1,4 +1,4 @@
-VHIVE_BRANCH='v1.7'
+VHIVE_BRANCH='main'
```

Comment:
Don't change the branch to main. We should either wait for new vhive version or negotiate a release of a new version.